openedx_events/tooling.py

```diff
@@ -185,8 +185,7 @@ def send_event(self, send_robust=True, time=None, **kwargs):
             [(<function callback at 0x7f2ce638ef70>, 'callback response')]
 
         Keyword arguments:
```

Comment:
[nit][not-blocking]
I find it confusing to have the arguments split up like this. Can we move this up with the other arguments?

Reply:
Sure! I was mimicking what I saw elsewhere in the file, but now that I check, that doesn't match our style guide. I'll fix it up.